### DISCUSSION TITLE
Are the plyr.io support mkv video format?

### DISCUSSION THREAD

**Discussion by 8KStudio** [sampotts/plyr#2812] [Q&A]
Lot of my video is using MKV format though

**Comment by sampotts** [comment]
Hey, mkv movies aren't web compatible; there are no codecs in browsers to play them back. 

https://github.com/sampotts/plyr/issues/2531
https://github.com/sampotts/plyr/issues/2391
https://github.com/sampotts/plyr/issues/2158
